### PR TITLE
plasma-applet-commandoutput: init at 2022-01-11

### PR DESCRIPTION
**Pull request by matthiasbeyer**
###### Motivation for this change

Want to try this.

CC @mdevlamynck because they packaged another plasma applet from this author and might be interested as well?

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/plasma-applet-commandoutput/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    repo = "plasma-applet-commandoutput";
+    rev = "d4c93a1fad82ff4d2fcc622f592b28a60878f49f";
+    sha256 = "0hg5i8yq159263yz7jyhpi893c5c2kl59w03p9nnpw656xr6102b";
+  };
+
+  # Adds the CMakeLists.txt not provided by upstream
+  patches = [ ./cmake.patch ];
+  postPatch = "rm build";
+  nativeBuildInputs = [ cmake extra-cmake-modules ];
+  buildInputs = [ plasma-framework kwindowsystem plasma-pa ];
```
Comment (mdevlamynck) [inline pkgs/applications/misc/plasma-applet-commandoutput/default.nix:18]: You probably don't need plasma-pa?

File: pkgs/applications/misc/plasma-applet-commandoutput/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  dontWrapQtApps = true;
+
+  meta = with lib; {
```
Comment (mdevlamynck) [inline pkgs/applications/misc/plasma-applet-commandoutput/default.nix:22]: You're missing the license field.

File: pkgs/applications/misc/plasma-applet-commandoutput/default.nix (hunk 1 above)
Comment (matthiasbeyer) [reply pkgs/applications/misc/plasma-applet-commandoutput/default.nix:18]: Ah, sure :laughing:

File: pkgs/applications/misc/plasma-applet-commandoutput/default.nix (hunk 2 above)
Comment (matthiasbeyer) [reply pkgs/applications/misc/plasma-applet-commandoutput/default.nix:22]: There is no license in the repository yet. I'll open an issue about that!

**Comment by matthiasbeyer** [comment]
Closing as I don't care enough and this has been open for way too long.

**Pull request closed without merging**